humanitec-resource-defs/redis/basic/variables.tf

```diff
@@ -0,0 +1,64 @@
+variable "humanitec_organization" {
```

Comment:
Where is this used?

Reply:
Good catch, removed. Will check if we can somehow check this in CI.